tutorials/installation-lemp-ubuntu-bionic/index.mdx

```diff
@@ -124,38 +127,45 @@ The [LEMP-Stack](https://lemp.io/) references a group of software that can be us
 
 4. Enter the new password and press `Enter`, then repeat this step.
 
-5. The installer will ask you some further questions. You can keep the setting proposed by default and validate it by pressing `Enter`.
+5. Keep the settings proposed by default when prompted by further questions, by pressing `Enter` each time.
 
 ## Configuring a Nginx Server-Block
 
 Nginx stores the configuration of virtual hosts in server blocks. All available server blocks are located in the directory `/etc/nginx/sites-available/`.
 
-1. Create a new server block configuration file and name it `example.com` as the following example:
+1. Navigate to the relevant directory:
+
+  ```
+  cd /etc/nginx/sites-available/
+  ```
+
+2. Create a new server block configuration file and name it after the domain name that points to your Instance. Here we use `example.com`:
 
-  ```nginx
+  ```
     server {
-            listen 80;
-            root /var/www/html;
-            index index.php index.html index.htm;
-            server_name example.com;
-
-            location / {
-                    try_files $uri $uri/ =404;
-            }
-
-            location ~ \.php$ {
-                    include snippets/fastcgi-php.conf;
-                    fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
-            }
-
-            location ~ /\.ht {
-                    deny all;
-            }
-    }
+          listen 80;
+          root /var/www/html;
+          index index.php index.html index.htm index.nginx-debian.html;
```

Comment:
```suggestion
          index index.php index.html index.htm;
```
I don't think you need the last file, it refers to the Debian specific index file only